test/tests/mortar/3d-periodic/periodic.i

```diff
@@ -59,7 +60,7 @@
     secondary_boundary = bottom
     primary_subdomain = 12
     secondary_subdomain = 11
-
+    delta = 0.1
```

Comment:
IIRC This is a stabilization parameter: can the name be more descriptive?

Reply:
This is the stabilization parameter but that's not something I implemented. I could change it but that might mess up people's tests and applications since it's been around for a while

Reply:
Ok then, never mind